core/nodepool/config/config.go

```diff
@@ -178,6 +178,8 @@ func (c *ProvidedConfig) Load(main *cfg.Config) error {
 	c.KubeClusterSettings = main.KubeClusterSettings
 	c.Experimental.TLSBootstrap = main.DeploymentSettings.Experimental.TLSBootstrap
 	c.Experimental.NodeDrainer = main.DeploymentSettings.Experimental.NodeDrainer
+	c.DeploymentSettings.CloudWatchLogging = main.DeploymentSettings.CloudWatchLogging
+	c.DeploymentSettings.JournaldCloudWatchLogsImage = main.DeploymentSettings.JournaldCloudWatchLogsImage
```

Comment:
I guess we'd better do these inside the `WithDeafultsFrom` func of `DeploymentSettings` which is called above: https://github.com/kubernetes-incubator/kube-aws/pull/733/files#diff-e564e8519cdc2363da40b406560eb023R175